packages/gatsby/src/bootstrap/load-plugins/__tests__/load-plugins.js

```diff
@@ -71,4 +71,76 @@ describe(`Load plugins`, () => {
 
     expect(plugins).toMatchSnapshot()
   })
+
+  describe(`TypeScript support`, () => {
+    it(`loads gatsby-plugin-typescript if not provided`, async () => {
+      const config = {
+        plugins: [],
+      }
+
+      let plugins = await loadPlugins(config)
+
+      plugins = replaceFieldsThatCanVary(plugins)
```

Comment:
Is it possible to create a different helper function here? The one that exists now also check paths property and might cause weird test failures when we're we might move more plugins as a default.

Reply:
I'm not sure. Do you have any suggestions? It seems to work right now.

Reply:
It's the same setup we have for `gatsby-source-page-creators`

Reply:
maybe `replaceFieldsThatCanVary` should also just do more checks before trying to do `pluginOptions.path` sanitization (apply it only for `gatsby-plugin-page-creator`)?

But this doesn't seem to be concern of this pull request?